package.json

```diff
@@ -41,7 +41,10 @@
     "npm-publish:rc": "lerna exec --no-private -- npm publish --access public --tag rc",
     "npm-publish:latest": "lerna exec --no-private -- npm publish --access public --tag latest",
     "npm-publish:next": "lerna exec --no-private -- npm publish --access public --tag next",
-    "npm-publish:canary": "lerna exec --no-private -- npm publish --access public --tag canary"
+    "npm-publish:canary": "lerna exec --no-private -- npm publish --access public --tag canary",
+    "changeset": "changeset",
+    "changeset:version": "changeset version; npm i",
```

Comment:
the `; npm i` part is because I noticed after `changeset version` while packages were updated, the main `lock` file wasn't so this fixes it